apps/wallet/src/features/Explorer/Missions/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { useRef } from 'react';
+import { FlatList, StyleSheet } from 'react-native';
+import { View } from '@walless/gui';
+
+import { missions } from '../internal';
+
+import MissionItem from './MissionItem';
+
+const Missions = () => {
+	const scrollRef = useRef<FlatList>(null);
+
+	return (
+		<View style={styles.container}>
+			<FlatList
```

Comment:
can you try using the `snapToOffsets` props of the `ScrollView` components to verify if it works on the web platform? That is the prop I saw from another project